src/nextjournal/clerk.clj

```diff
@@ -508,6 +508,18 @@
         {state :result duration :time-ms} (time-ms (write-static-app! opts state))]
     (report-fn {:stage :finished :state state :duration duration :total-duration (elapsed-ms start)})))
 
+(defn render-fn* [sym]
+  (let [ns (namespace sym)
+        file (str (munge ns) ".cljs")]
```

Comment:
I'm trying this with a symbol `scratch.freehand/render-fn` (file lives in `notebooks/scratch/freehand.cljs`) munge alone doesn't convert `.` in `/` right? Or do we make some assumptions on the location of the cljs files?

Reply:
Good catch! We have to convert `"."` to `"/"`, I will push that.